src/coreclr/ilasm/asmparse.y

```diff
@@ -42,13 +42,13 @@
 };
 
         /* These are returned by the LEXER and have values */
-%token ERROR_ BAD_COMMENT_ BAD_LITERAL_                         /* bad strings,    */
```

Comment:
This has been defined multiple times, so remove it here.

Reply:
Makes sense to me.